### ISSUE TITLE
This custom component will stop working in Home Assistant 2022.6

### ISSUE THREAD

**Issue by jamespreedy** [mvdwetering/huesyncbox#26]
Hi there, thanks for your work on this custom component.

This custom component uses discovery_info['host'] instead of discovery_info.host.
This will fail in Home Assistant version 2022.6
The Home Assistant core logs asked me to report this issue to the custom component author.
Please see below.

`Logger: homeassistant.helpers.frame
Source: helpers/frame.py:103 
First occurred: 9:39:19 am (1 occurrences) 
Last logged: 9:39:19 am

Detected integration that accessed discovery_info['host'] instead of discovery_info.host; this will fail in version 2022.6. Please report issue to the custom component author for huesyncbox using this method at custom_components/huesyncbox/config_flow.py, line 65: "host": discovery_info["host"],`

Please let me know if you need any additional information, or I can help at all.
Thanks in advance!

**Comment by gvssr** [comment]
Seeing the same message here. Also:

```
Detected integration that accessed discovery_info['host'] instead of discovery_info.host; this will fail in version 2022.6. Please report issue to the custom component author for huesyncbox using this method at custom_components/huesyncbox/config_flow.py, line 65: "host": discovery_info["host"],
```

**Comment by kfran78** [comment]
Same in my side, specially when i reboot my host and/or HA
Detected integration that accessed discovery_info['host'] instead of discovery_info.host; this will fail in version 2022.6. Please report issue to the custom component author for huesyncbox using this method at custom_components/huesyncbox/config_flow.py, line 65: "host": discovery_info["host"],

**Comment by kfran78** [comment]
> Hi there, thanks for your work on this custom component.
> 
> This custom component uses discovery_info['host'] instead of discovery_info.host. This will fail in Home Assistant version 2022.6 The Home Assistant core logs asked me to report this issue to the custom component author. Please see below.
> 
> `Logger: homeassistant.helpers.frame Source: helpers/frame.py:103 First occurred: 9:39:19 am (1 occurrences) Last logged: 9:39:19 am
> 
> Detected integration that accessed discovery_info['host'] instead of discovery_info.host; this will fail in version 2022.6. Please report issue to the custom component author for huesyncbox using this method at custom_components/huesyncbox/config_flow.py, line 65: "host": discovery_info["host"],`
> 
> Please let me know if you need any additional information, or I can help at all. Thanks in advance!

But you keep your integration and it continue to work or you desactivate it ?

**Comment by jamespreedy** [comment]
> > Hi there, thanks for your work on this custom component.
> > This custom component uses discovery_info['host'] instead of discovery_info.host. This will fail in Home Assistant version 2022.6 The Home Assistant core logs asked me to report this issue to the custom component author. Please see below.
> > `Logger: homeassistant.helpers.frame Source: helpers/frame.py:103 First occurred: 9:39:19 am (1 occurrences) Last logged: 9:39:19 am
> > Detected integration that accessed discovery_info['host'] instead of discovery_info.host; this will fail in version 2022.6. Please report issue to the custom component author for huesyncbox using this method at custom_components/huesyncbox/config_flow.py, line 65: "host": discovery_info["host"],`
> > Please let me know if you need any additional information, or I can help at all. Thanks in advance!
> 
> But you keep your integration and it continue to work or you desactivate it ?

I keep it and it continues to work. I believe from the error that this will only stop working in 2022.6.

**Comment by mvdwetering** [comment]
Thanks for the heads up.
Will have a look at it

**Comment by mvdwetering** [comment]
This has been solved in v1.16.0
